app/mailers/hypertension_fast_track_pilot_mailer.rb

```diff
@@ -5,7 +5,7 @@ def build(submission)
     @submission = submission
 
     subject =
-      if submission.form_json.include? 'VAMC_Hypertension_Rapid_Decision_Evidence.pdf'
+      if submission.form_json.include? 'VAMC_Hypertension_Rapid_Decision_Evidence'
```

Comment:
I wonder if this could be made a constant just so that name itself isn't floating around